src/main/java/com/hms_networks/sc/cumulocity/CConnectorMain.java

```diff
@@ -565,7 +654,7 @@ public static boolean runProvisioningMqtt() {
               + connectorConfig.getCumulocityBootstrapUsername();
       final String mqttPassword = connectorConfig.getCumulocityBootstrapPassword();
       final String mqttRootCaFilePath = CConnectorApiCertificateMgr.getRootCaFilePath();
-      CConnectorProvisionMqttMgr provisioningMqttMgr =
+      provisioningMqttMgr =
```

Comment:
I thought java was statically typed.

Reply:
Not sure what you mean, but this variable declaration was just moved to be at the class level so that the various shutdown/restart methods can cancel provisioning if it is in progress. Otherwise, the connector hangs if it receives a shutdown/restart request while in provisioning MQTT.

Reply:
https://github.com/hms-networks/flexy-cumulocity-connector/blob/29ecce3e9b19b8a4a1a830103417e5f6640becc1/src/main/java/com/hms_networks/sc/cumulocity/CConnectorMain.java#L93-L97

Reply:
I didn't see that the declaration was moved.